### PR TITLE
Add loggers, remove traces of persons in output + minor code stuff

### PR DESCRIPTION
**Pull request by sheehui**

### CODE REVIEW

**Review by mrmrinal — APPROVED** [review]
LGTM! Nothing ever seems to escape your eye Inspector!

File: src/main/java/donnafin/model/person/UniquePersonList.java [hunk 1]
```diff
@@ -14,9 +14,9 @@
 /**
  * A list of persons that enforces uniqueness between its elements and does not allow nulls.
  * A person is considered unique by comparing using {@code Person#isSamePerson(Person)}. As such, adding and updating of
- * persons uses Person#isSamePerson(Person) for equality so as to ensure that the person being added or updated is
- * unique in terms of identity in the UniquePersonList. However, the removal of a person uses Person#equals(Object) so
- * as to ensure that the person with exactly the same fields will be removed.
+ * persons uses Person#isSamePerson(Person) for equality to ensure that the person being added or updated is
+ * unique in terms of identity in the UniquePersonList. However, the removal of a person uses Person#equals(Object)
+ * to ensure that the person with exactly the same fields will be removed.
```
Comment (bharathcs) [inline src/main/java/donnafin/model/person/UniquePersonList.java:19]:
```suggestion
 * persons uses {@code Person#isSamePerson(Person)} for equality to ensure that the person being added or updated
 * is unique in terms of identity in the {@code UniquePersonList}. However, the removal of a person uses
 * {@code Person#equals(Object)} to ensure that the person with exactly the same fields will be removed.
```

**Review by bharathcs — DISMISSED** [review]
Niceee. glad you've done the logger work, and the logo is a nice touch.

File: src/main/java/donnafin/model/person/UniquePersonList.java [hunk 2]
```diff
@@ -14,9 +14,9 @@
 /**
  * A list of persons that enforces uniqueness between its elements and does not allow nulls.
  * A person is considered unique by comparing using {@code Person#isSamePerson(Person)}. As such, adding and updating of
- * persons uses Person#isSamePerson(Person) for equality so as to ensure that the person being added or updated is
- * unique in terms of identity in the UniquePersonList. However, the removal of a person uses Person#equals(Object) so
- * as to ensure that the person with exactly the same fields will be removed.
+ * persons uses {@code Person#isSamePerson(Person)} for equality to ensure that the person being added or updated
+ * is unique in terms of identity in the {@code UniquePersonList}. However, the removal of a person uses {@code Person#equals(Object)}
+ * to ensure that the person with exactly the same fields will be removed.
```
Comment (bharathcs) [inline src/main/java/donnafin/model/person/UniquePersonList.java:19]:
```suggestion
 * persons uses {@code Person#isSamePerson(Person)} for equality to ensure that the person being added or updated
 * is unique in terms of identity in the {@code UniquePersonList}. However, the removal of a person uses
 * {@code Person#equals(Object)} to ensure that the person with exactly the same fields will be removed.
```

**Review by bharathcs — APPROVED** [review]
LGTM!!

**Pull request merged**